styles/build.scss

```diff
@@ -0,0 +1 @@
+@import 'angular-patternfly.scss';
```

Comment:
@dtaylor113 can you remove the `.scss` extension from this import statement so that all of the imports are standardized?

Reply:
ok, done